### ISSUE TITLE
Add a license

### ISSUE THREAD

**Issue by saracope** [wgxcodersdc/front-end-community#30]
I think this is a great resource and I'd like to start a similar front end lab night in my community (Dayton, Ohio). I would appreciate if there was a license attached to the repo to know if the materials are open source or not. Thanks!

**Comment by mmlumba** [comment]
Yes, feel free to use the resources; just make sure to give credit. I'll add a license to the repo shortly.

**Comment by JlaPrs** [comment]
Hey, I don't even know which resource I'm using from you that I should give
you a credit. For what exactly?

Best wishes

Marian Lumba <notifications@github.com> schrieb am Do., 25. Apr. 2019,
18:38:

**Comment by mmlumba** [comment]
My above comment was for @saracope , so please ignore. I thought github issues only notifies those who have contributed to the issue; not sure how many others this issue has notified

**Comment by saracope** [comment]
@mmlumba Thanks!
